### PR TITLE
Revert getting token endpoint from well known config

### PR DESCRIPTION
**Pull request by shahzaibj**
This to fix the issue caused by Blackforest migrating to Public cloud as the well known config still returns the DE token endpoint for Blackforest cloud even after migration.

Also the other reason to revert this is to avoid potential simulated instance aware in the case of a tenanted authority.

The other reason to revert is because we don't know what would be expected behavior once cross cloud is implemented.

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Version 2.1.1
 ----------
 - Introduces result sharing to minimize duplicate_command errors
-- Device Code authorization endpoint is now sourced from .Well-Known config
+- No longer query well known config to obtain token endpoint - build it manually instead
```
Comment (AdamBJohnsonx) [inline changelog.txt:4]: I mean, if this is a revert, then nothing changes, right?

Comment (shahzaibj) [reply changelog.txt:4]: Nothing changes for Device Code flow, hence I removed that line. But I added a new line about reverting using well known config for Token Endpoint as that change was made back in November 2019 here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/714

**Comment by shahzaibj** [comment]
> PMD failed
> 
> :(

Fixed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1015/commits/5568d59f42e987c5f7a191f68b9c300bd5752c10

File: common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Configuration.java [hunk 2]
```diff
@@ -52,22 +52,15 @@
      * @return URL the authorization endpoint
      */
     public URL getAuthorizationEndpoint() {
-        final OpenIdProviderConfiguration openIdConfig = getOpenIdWellKnownConfigForAuthority();
-        if (openIdConfig != null) {
-            return getEndpointUrlFromAuthority(openIdConfig.getAuthorizationEndpoint());
-        }
         return getEndpointUrlFromRootAndSuffix(getAuthorityUrl(), FALLBACK_AUTHORIZE_ENDPOINT_SUFFIX);
```
Comment (kreedula) [inline common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Configuration.java:55]: Rename to `AUTHORIZE_ENDPOINT_SUFFIX` ? and similar below

Comment (shahzaibj) [reply common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Configuration.java]: Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1015/commits/161957ecc585b2fb06601d6ff19c96df7f3d494c

**Pull request merged**